### ISSUE TITLE
Post-release UI cleanups 

### ISSUE THREAD

**Issue by gcampax** [stanford-oval/almond-cloud#985]
After #977, there are some left-over UI issues that are worth polishing:

- font sizes are inconsistent and often wrong (see for example in Commandpedia, secondary text is bigger than primary text)
- colored and plain buttons are inconsistent, for example in Thingpedia device pages
- the X button in the conversation widget is broken when the mic button is used
- spacing in the "My Skill" panel is inconsistent

We should clean up all of these, apply them to the stable branch, and release as 2.0.1 soon.
A separate issue will be filed to track major UI changes for the next development cycle.
